### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by ZhijunZhao**
@yungez @xscript @yaweiw

### CODE REVIEW

File: azure-spring-boot-starters/azure-documentdb-spring-boot-starter/README.md [hunk 1]
```diff
@@ -130,6 +130,6 @@ Find more information about Azure Service Privacy Statement, please check [Micro
 
 ### Further info
 
-Besides using Azure Cosmos DB DocumentDB API Spring boot starter, you can directly use Azure Cosmos DB DocumentDB API Spring Data package to more complex scenario, detail pls refer to [Spring Data for Azure Cosmos DB DocumentDB API](https://github.com/Microsoft/spring-data-documentdb).
+Besides using Azure Cosmos DB DocumentDB API Spring boot starter, you can directly use Azure Cosmos DB DocumentDB API Spring Data package to more complex scenario, detail please refer to [Spring Data for Azure Cosmos DB DocumentDB API](https://github.com/Microsoft/spring-data-documentdb).
```
Comment (yaweiw) [inline azure-spring-boot-starters/azure-documentdb-spring-boot-starter/README.md]: detail please refer to [xxx] ===> please refer to [xxxx] for more details.

File: azure-spring-boot-starters/azure-keyvault-secrets-spring-boot-starter/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ## Azure Key Vault Secrets Spring boot starter
-Azure Key Vault Secrets Spring boot starter is Spring starter for [Azure Key Vault Secrets](https://docs.microsoft.com/en-us/rest/api/keyvault/about-keys--secrets-and-certificates#BKMK_WorkingWithSecrets). With this starter, Azure Key Vault is added as one of Spring PropertySource, so secrets stored in Azure Key Vault could be used easily and conviniently accessed like other externalized configuration property, e.g. properties in files.
+Azure Key Vault Secrets Spring boot starter is Spring starter for [Azure Key Vault Secrets](https://docs.microsoft.com/en-us/rest/api/keyvault/about-keys--secrets-and-certificates#BKMK_WorkingWithSecrets). With this starter, Azure Key Vault is added as one of Spring PropertySource, so secrets stored in Azure Key Vault could be used easily and conveniently accessed like other externalized configuration property, e.g. properties in files.
```
Comment (yaweiw) [inline azure-spring-boot-starters/azure-keyvault-secrets-spring-boot-starter/README.md]: used easily and conveniently accessed => easily used and conveniently accessed

File: HowToContribute.md [hunk 3]
```diff
@@ -11,31 +11,31 @@ To build the project, run maven commands.
...
 ```bash
-mvnw clean install
+mvn clean install
 ```
 
-- Skip tests execution
+- Skip test execution
 ```bash
-mvnw clean install -DskipTests
+mvn clean install -DskipTests
```
Comment (rajadilipkolli) [inline HowToContribute.md]: I think mvnw is still valid, any reasons to change it back to mvn?

Comment (ZhijunZhao) [reply HowToContribute.md]: On windows, `mvnw` is fine. On mac, it has to be `./mvnw`, for `mvnw` is not recognizable. I don't want to complicate these commands by differentiating platforms.

**Pull request merged**